internal/auth/credentials.go

```diff
@@ -140,7 +140,8 @@ func (cred Credentials) IsTemp() bool {
 
 // IsServiceAccount - returns whether credential is a service account or not
 func (cred Credentials) IsServiceAccount() bool {
-	return cred.ParentUser != "" && (cred.Expiration.IsZero() || cred.Expiration.Equal(timeSentinel))
+	_, ok := cred.Claims["sa-policy"]
```

Comment:
```suggestion
	_, ok := cred.Claims[iamPolicyClaimNameSA()]
```

Reply:
iamPolicyClaimNameSA() is defined in `cmd` pkg. Declaring a const here in `auth` pkg to avoid naked strings